src/config.rs

```diff
@@ -378,7 +378,7 @@ impl Config {
         } else {
             stderrlog::Timestamp::Off
         });
-        log.init().expect("logging initialization failed");
```

Comment:
Why?

Reply:
At some point of searching for working tracing implementation this was needed to remove runtime error. I think something else was already initialising log. Anyway in current version this is not a problem any more. I restored this line.